test/fixtures/sitemap.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
     <url>
         <loc>https://hex.pm/packages/postgrex</loc>
-        <lastmod>2014-04-17T14:00:00Z</lastmod>
+        <lastmod>2014-04-17T14:00:00</lastmod>
```

Comment:
@ericmj Per https://github.com/elixir-lang/ecto/pull/1216, Ecto no longer assumes that date is stored in UTC, however we do so in HexWeb so I brought back the "Z" here: 6ab28b6906c580ac71a9a7c9bd0e7b94bef187b8